airbyte-integrations/connectors/source-oura/source_oura/oura.yaml

```
@@ -0,0 +1,118 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["data"]
+  base_requester:
+    url_base: "https://api.ouraring.com/v2/usercollection"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+  date_requester:
+    $ref: "*ref(definitions.base_requester)"
+    request_options_provider:
+      request_parameters:
+        start_date: "{{ config['start_date'] }}"
+        end_date: "{{ config['end_date'] }}"
```

```suggestion
        end_date: "{{ config['end_date'] or utc_now() }}"
```
Check more about macros here: https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/macros.py

---

This is the API default anyway, so I'm not sure this is necessary.